frontend/packages/console-app/src/components/resource-quota/AppliedClusterResourceQuotaCharts.tsx

```diff
@@ -20,9 +20,9 @@ const AppliedClusterResourceQuotaCharts = ({
   );
 
   const charts = Object.keys(nsQuotas?.status?.hard ?? {}).map((resourceName) => {
-    const clusterHard = appliedClusterResourceQuota.status.total?.hard?.[resourceName];
-    const clusterUsed = appliedClusterResourceQuota.status.total?.used?.[resourceName];
-    const nsUsed = nsQuotas.status.used?.[resourceName];
+    const clusterHard = appliedClusterResourceQuota.status?.total?.hard?.[resourceName] ?? '';
+    const clusterUsed = appliedClusterResourceQuota.status?.total?.used?.[resourceName] ?? '';
+    const nsUsed = nsQuotas?.status?.used?.[resourceName] ?? '';
```

Comment:
We can get rid of these default values by making the `used` and `hard` props optional in [getUsedPercentage](https://github.com/openshift/console/blob/main/frontend/packages/console-app/src/components/resource-quota/utils.ts#L16) and [getLabelAndUsage](https://github.com/openshift/console/blob/main/frontend/packages/console-app/src/components/resource-quota/utils.ts#L29), since they are optional in the parent components."